util/rlimit_other.go

```diff
@@ -23,12 +23,12 @@ import (
 )
 
 // GenRLimit get RLIMIT_NOFILE limit
-func GenRLimit() uint64 {
+func GenRLimit(source string) uint64 {
 	rLimit := uint64(1024)
 	var rl syscall.Rlimit
 	err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rl)
 	if err != nil {
-		logutil.BgLogger().Warn("[ddl-ingest] get system open file limit error", zap.Error(err), zap.String("default", "1024"))
+		logutil.BgLogger().Warn("get system open file limit error", zap.Error(err), zap.String("source", source), zap.String("default", "1024"))
```

Comment:
Maybe we can update it like `[%s] get system open file limit error`?
Because I remember a lot of logs have `[ddl-ingest]` in them.

Reply:
> Maybe we can update it like `[%s] get system open file limit error`? Because I remember a lot of logs have `[ddl-ingest]` in them.

Ok